src/test/kotlin/pt/up/fe/ni/website/backend/controller/ProjectControllerTest.kt

```diff
@@ -868,6 +898,140 @@ internal class ProjectControllerTest @Autowired constructor(
         }
     }
 
+    @NestedTest
+    @DisplayName("PUT /projects/{idProject}/addHallOfFameMember/{idAccount}")
+    inner class AddHallOfFameMember {
+        private val newAccount = Account(
+            "Another test Account",
+            "test3_account@test.com",
+            "test_password",
+            "This is another test account too",
+            TestUtils.createDate(2002, Calendar.JULY, 1),
+            "https://test-photo.com",
+            "https://linkedin.com",
+            "https://github.com",
+            listOf(
+                CustomWebsite("https://test-website.com", "https://test-website.com/logo.png")
+            )
+        )
+
+        @BeforeEach
+        fun addToRepositories() {
+            accountRepository.save(testAccount)
+            accountRepository.save(testAccount2)
+            accountRepository.save(newAccount)
+            repository.save(testProject)
+        }
+
+        private val parameters = listOf(
+            parameterWithName("idProject").description(
+                "ID of the project to add the account to the hall of fame"
```

Comment:
Something along these lines would be clearer imo
```suggestion
                "ID of the project whose hall of fame the account will be added to"
```